styles.css

```diff
@@ -41,16 +41,41 @@
   align-items: center;
 }
 .menu__item {
-  padding: 0 13px;
+  position: relative;
+  margin-right: 26px;
+  font: 18px/21px roboto, helvetica, arial, sans-serif;
+  color: #000;
+  text-decoration: none;
+  transition: opacity .5s width .5s;
 }
-.menu__item:last-child {
-  padding-right: 0;
+.menu__item_active::after {
+ position: absolute;
+ display: block;
+ content: '';
+ width: 100%;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/HTML_SNN/pullRequest?prid=6243807)